coax/policy_objectives/_soft_pg.py

```diff
@@ -54,4 +55,15 @@ def objective_func(self, params, state, hyperparams, rng, transition_batch, Adv)
         chex.assert_rank([W, Q], 1)
         objective = W * Q
 
+        if type(self.regularizer) == NStepEntropyRegularizer:
+            if type(transition_batch.extra_info) != dict:
```

Comment:
Please could you use `isinstance` for type checking? See e.g. [this post](https://stackoverflow.com/questions/1549801/what-are-the-differences-between-type-and-isinstance).